### PR TITLE
1127 Permissions Checking: GET Survey Response

### PR DESCRIPTION
**Pull request by EMcQ-BES**
### Changes:

- Added GETHandler class for survey response requests
- Currently runs permissions for singular requests

### CODE REVIEW

File: packages/meditrak-server/src/permissions/get/hasSurveyResponsePermissions.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+/**
```
[the remark this answers is not in the document]
Comment (EMcQ-BES) [reply packages/meditrak-server/src/permissions/get/hasSurveyResponsePermissions.js:1]: I've made these changes on the other PR https://github.com/beyondessential/tupaia/pull/1190

Just to keep merges clean

File: packages/meditrak-server/src/routes/GETSurveyResponses.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    // Is there a way to check they have any permission group that may contain survey responses?
+    return this.assertPermissions(allowNoPermissions);
+  }
+
+  async findSingleRecord(surveyResponseId, options) {
+    const surveyResponse = await this.database.findOne(
+      this.recordType,
+      { [`${this.recordType}.id`]: surveyResponseId },
+      options,
+    );
```
[the remark this answers is not in the document]
Comment (EMcQ-BES) [reply packages/meditrak-server/src/routes/GETSurveyResponses.js:30]: I've made this change in the other PR https://github.com/beyondessential/tupaia/pull/1190

Just so things will merge cleany.

File: packages/meditrak-server/src/app/addRoutesToApp.js [hunk 3]
```diff
@@ -101,6 +102,7 @@ export function addRoutesToApp(app) {
   app.get('(/v[0-9]+)/me/countries', getCountryAccessList);
   app.get('(/v[0-9]+)/answer/:recordId?', getAnswers);
   app.get('(/v[0-9]+)/disaster/:recordId?', getDisasters);
+  app.get('(/v[0-9]+)/surveyResponse/:recordId?', getSurveyResponses);
```
Comment (chris-bes) [inline packages/meditrak-server/src/app/addRoutesToApp.js:105]: I've also just realized that this should probably be `surveyResponses` as that's how the url the Admin Panel will send

Comment (EMcQ-BES) [reply packages/meditrak-server/src/app/addRoutesToApp.js:105]: Fixed

**Review by chris-bes — APPROVED** [review]
There are some merge conflicts but pre approving

**Pull request merged**